packages/accordion/README.md

```diff
@@ -1,13 +1,10 @@
-# &lt;vaadin-accordion&gt;
+# @vaadin/accordion
 
-[&lt;vaadin-accordion&gt;](https://vaadin.com/components/vaadin-accordion) is a Web Component implementing the vertically stacked set of expandable panels, part of the [Vaadin components](https://vaadin.com/components).
+A web component implementing accordion widget: a vertically stacked set of expandable panels.
```

Comment:
```suggestion
A web component for displaying a vertically stacked set of expandable panels.
```

Widget is not a good term here, also it's obvious that it's about an accordion.

Reply:
Thanks, changed.